.github/workflows/branch-release.yml

```
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2025 Canonical Ltd.
+name: Branch Release
+
+on:
+  workflow_run:
+    workflows: ["Tag GitHub"]
```

What about something like this
```suggestion
    workflows:
        - "Tag GitHub"
```

---

Done